src/main/java/pl/javastart/task/Main.java

```diff
@@ -10,7 +16,47 @@ public static void main(String[] args) {
     }
 
     public void run(Scanner scanner) {
-        // uzupełnij rozwiązanie. Korzystaj z przekazanego w parametrze scannera
+        DateTimeFormatter formatter = getDateTimeFormatter();
+        System.out.println("Podaj datę:");
+        String userDate = scanner.nextLine();
+        TemporalAccessor parse = formatter.parse(userDate);
+        LocalDateTime localTime = LocalDateTime.from(parse);
```

Comment:
Tutaj postaraj się wczytać i mieć już ZonedDateTime, a nie LocalDateTime. Zoned ma funkcjonalności związane z przeliczaniem stref czasowy, a o to przecież chodzi w zadaniu